i18n/en-US.properties

```diff
@@ -120,6 +120,8 @@ be.contentSidebar.activityFeed.commentForm.approvalDueDate = Due Date
 be.contentSidebar.activityFeed.commentForm.approvalSelectDate = Select a date
 # Mentioning call to action displayed below the comment input
 be.contentSidebar.activityFeed.commentForm.atMentionTip = @mention users to notify them.
+# Mentioning call to action detailed description for screen reader users
+be.contentSidebar.activityFeed.commentForm.atMentionTipDescription = Use the @ symbol to mention users, and use the up and down arrow keys to scroll through autocomplete suggestions.
```

Comment:
Can we either remove the comma after `users` and/or break this into two sentences?